book.json

```diff
@@ -53,7 +53,7 @@
                     "label": "Edit page on github",
                     "icon": "fa fa-pencil-square-o",
                     "position" : "left",
-                    "url": "https://github.com/mavlink/MAVSDK-docs/edit/develop/{{filepath_lang}}"
+                    "url": "https://github.com/mavlink/MAVSDK-docs/edit/main/{{filepath_lang}}"

```

Comment:
Ah, so we develop on main now. That makes a lot more sense to me.

Reply:
Yes, since we never backported fixes anyway, there is no need for the complication.